Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gemspec
 
 gem "administrate-field-image"
+gem "bourbon"
```

Comment:
@teoljungberg Shall we lock this down to `4.x`, `, "~> 4.2"`? Once 5 ships, it will bust.